src/interfaces/IMorpho.sol

```diff
@@ -123,6 +123,8 @@ interface IMorphoBase {
     /// `liquidate` from being used under certain market conditions.
     /// - A very high price returned by the oracle can make the computation of `maxBorrow` in `_isHealthy` overflow, or
     /// the computation of `assetsRepaid` in `liquidate` overflow.
+    /// @dev The share price of a market with less than 1e4 assets borrowed can be decreased by manipulations, to the
+    /// point where totalBorrowShares is very large and borrowing overflows.
```

Comment:
```suggestion
    /// @dev When there are less than 1e4 assets borrowed in a market, their share price can be decreased by
    /// manipulations, to the point where `totalBorrowShares` is very large and borrowing overflows.
```
because share price can also refer to the share price of supply assets

Reply:
```suggestion
    /// @dev The borrow share price of a market with less than 1e4 assets borrowed can be decreased by manipulations, to the
    /// point where totalBorrowShares is very large and borrowing overflows.
```